tests/legacy-cli/e2e_runner.ts

```diff
@@ -49,6 +53,7 @@ const argv = yargsParser(process.argv.slice(2), {
   ],
   string: ['devkit', 'glob', 'ignore', 'reuse', 'ng-tag', 'tmpdir', 'ng-version'],
   number: ['nb-shards', 'shard'],
+  array: ['package'],
```

Comment:
For the time being can we default this to `./dist/_*.tgz`? The command itself is already a bit long and having to write `--package=/dist/_*.tgz` multiple times a day seems a bit unnecessary.

For context here's an example of the command now.
```
node tests/legacy-cli/run_e2e.js tests/legacy-cli/e2e/tests/build/library/lib-consumption-full-aot.ts --package=/dist/_*.tgz
```